### DISCUSSION TITLE
Biometrics: capacitor-native-biometric package don't work with Android (capacitor 5)

### DISCUSSION THREAD

**Discussion by crsimoes** [ionic-team/capacitor#6950] [Q&A]
capacitor-native-biometric it is not working with Android (iOS is working)
 
 We are using Capacitor 5.

Everytime that we use capacitor-native-biometric with Android we have the following error:

 D  Sending plugin error: {"save":false,"callbackId":"57102556","pluginId":"NativeBiometric","methodName":"getCredentials","success":false,"error":{"message":"No credentials found"}}
V  To native (Capacitor plugin): callbackId: 57102557, pluginId: NativeBiometric, methodName: setCredentials
android.security.keystore2.KeyStoreCryptoOperationUtils.getInvalidKeyException(KeyStoreCryptoOperationUtils.java:128)

dependencies:
"@capacitor/app": "^5.0.0",
"@capacitor/core": "^5.4.1",
"@capacitor/android": "^5.4.1",
"@capacitor/ios": "^5.4.1",
"@capacitor/cli": "^5.4.1",

**Comment by shahrirosm** [comment]
checkout this plugin https://github.com/aparajita/capacitor-biometric-auth
